drivers/eaton-ats30-mib.c

```diff
@@ -0,0 +1,370 @@
+	/* enterprises.534.10.1.2.2.0 = STRING: "1A0003AR00.00.00" -- Firmware */
+	{ "ups.firmware", ST_FLAG_STRING, SU_INFOSIZE, ".1.3.6.1.4.1.534.10.1.2.2.0", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.2.3.0 = STRING: "2014-09-17      "  -- Release date */
+	/* { "unmapped.enterprises", ST_FLAG_STRING, SU_INFOSIZE, ".1.3.6.1.4.1.534.10.1.2.3.0", NULL, SU_FLAG_OK, NULL, NULL }, */
+	/* enterprises.534.10.1.2.4.0 = STRING: "JA00E52021          " */
+	{ "device.serial", ST_FLAG_STRING, SU_INFOSIZE, ".1.3.6.1.4.1.534.10.1.2.4.0", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.2.5.0 = STRING: "                    " -- Device ID codes */
+	/* { "unmapped.enterprises", ST_FLAG_STRING, SU_INFOSIZE, ".1.3.6.1.4.1.534.10.1.2.5.0", NULL, SU_FLAG_OK, NULL, NULL }, */
+    
+    /* ats measure */
+    /* =========== */
+	/* enterprises.534.10.1.3.1.1.1.1 = INTEGER: 1 */
+	{ "input.1.id", 0, 1, ".1.3.6.1.4.1.534.10.1.3.1.1.1.1", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.1.1.1.2 = INTEGER: 2 */
+	{ "input.2.id", 0, 1, ".1.3.6.1.4.1.534.10.1.3.1.1.1.2", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.1.1.2.1 = INTEGER: 2379 */
+	{ "input.1.voltage", 0, 0.1, ".1.3.6.1.4.1.534.10.1.3.1.1.2.1", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.1.1.2.2 = INTEGER: 0 */
+	{ "input.2.voltage", 0, 0.1, ".1.3.6.1.4.1.534.10.1.3.1.1.2.2", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.1.1.3.1 = INTEGER: 500 */
+	{ "input.1.frequency", 0, 0.1, ".1.3.6.1.4.1.534.10.1.3.1.1.3.1", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.1.1.3.2 = INTEGER: 0 */
+	{ "input.2.frequency", 0, 0.1, ".1.3.6.1.4.1.534.10.1.3.1.1.3.2", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.2.1.0 = INTEGER: 2375 */
+	{ "output.voltage", 0, 0.1, ".1.3.6.1.4.1.534.10.1.3.2.1.0", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.2.2.0 = INTEGER: 0 */
+	{ "output.current", 0, 0.1, ".1.3.6.1.4.1.534.10.1.3.2.2.0", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.3.0 = INTEGER: 25 -- internal temperature in celsius */
+	{ "ups.temperature", 0, 1, ".1.3.6.1.4.1.534.10.1.3.3.0", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.4.0 = INTEGER: 77 -- internal temperature in farenhait */
+	/* { "ups.temperetureF", 0, 1, ".1.3.6.1.4.1.534.10.1.3.4.0", NULL, SU_FLAG_OK, NULL, NULL }, */
+	/* enterprises.534.10.1.3.5.0 = INTEGER: 37937541 */
+	{ "device.uptime", 0, 1, ".1.3.6.1.4.1.534.10.1.3.5.0", NULL, SU_FLAG_OK, NULL, NULL },
+	/* enterprises.534.10.1.3.6.0 = INTEGER: 284 */
+	/* { "unmapped.atsMessureTransferedTimes", 0, 1, ".1.3.6.1.4.1.534.10.1.3.6.0", NULL, SU_FLAG_OK, NULL, NULL }, */
+	/* enterprises.534.10.1.3.7.0 = INTEGER: 4 */
+	{ "input.source", 0, 1, ".1.3.6.1.4.1.534.10.1.3.7.0", NULL, SU_FLAG_OK, ats30_source_info, NULL },
+    
+    /* atsStatus */
+    /* ========= */
+#if 0
```

Comment:
Should those `#if 0` blocks stay? If there is a possibility of uncommenting them later (e.g. when more research/discovery/support is done, then at least a comment is IMHO due to explain what the hard ifdef is for and why the code is just not deleted.